### ISSUE TITLE
Target size of buttons "Clear extension" and "Filter extension" is 22X22 which is less than required size 24x24pixel: A11y_VS Code_Extension_Target Size

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#222103]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #SH_WCAG2.2_Visual Studio Code Client_Win32_April2024; #Visual Studio Code Client; #Win32; #Win11; #WCAG2.5.8; #FTP; #A11yWCAG2.2; #TargetSize;

### Environment and OS details:
Version: 1.92.0-insider (user setup)
Windows 11 pro Version 23H2
OS Build: 22631.3672

### Repro Steps:

1. Open Visual Studio Code Insiders editor.
2. Tab till extension and press enter.
3. Tab till clear extension button and press enter.
4. Observed that Target size of buttons "Clear extension" and "Filter extension" is 22X22 which is less than required size 24x24pixel.

### Actual Result:
Target size of buttons "Clear extension" and "Filter extension" is 22X22 which is less than required size 24x24pixel.

### Expected Result:
Target size of buttons "Clear extension" and "Filter extension" should be minimum 24x24pixel. If the controls have spaced in-between, then those spaces should not intersect each other and target size with spacing should be 24x24 pixel for each one.

### User Impact:
Users with dexterity limitations and those who have difficulty with fine motor movement find it difficult to accurately activate small targets when there are other targets that are too close. Providing sufficient size, or sufficient spacing between targets, will reduce the likelihood of accidentally activating the wrong control.

### Attachment:

https://github.com/user-attachments/assets/0e931192-bafc-4ee1-9654-ca241f91eadc

Ruler Attachment:

https://github.com/user-attachments/assets/1e89adfd-23c4-4452-b8e6-cc6196899bb6

**Comment by isidorn** [comment]
No plans to fix. Latest findings from the test team is that the actual size of our buttons is ok.
Also this will only be a requirement in 2025, so no need to track this yet. Thus closing

**Comment by kkbrooks** [comment]
@meganrogge I'm starting to look into this, when does the requirement go into effect?

**Comment by meganrogge** [comment]
@AnKushSingh05 what is the deadline for this?

**Comment by kkbrooks** [comment]
@AnKushSingh05 what is the deadline for addressing this new requirement?

**Comment by AnKushSingh05** [comment]
Downgrading the bug to sev3 as width=22px

![Image](https://github.com/user-attachments/assets/18daa98c-eb4f-48dd-a164-8283a9a5aef8)

**Comment by AnKushSingh05** [comment]
Closing the bug as after zooming buttons are passing the target size

![Image](https://github.com/user-attachments/assets/a1133871-1870-4d90-900a-09bdc917de51)

![Image](https://github.com/user-attachments/assets/d60fb315-8565-4b70-b767-a8f87b02cc41)
